.github/workflows/main.yml

```
@@ -1,10 +1,10 @@
 name: CI for RIF-Relay-Contract
 on: [push]
 jobs:
-    lint:
+    lint&test:
```

```suggestion
    lint_and_test:
```
You can't use & in a job name.

---

Also why are we combining both tests in a single job?

---

thought it makes sense to combine them, since the tests are dependent on successful linting anyway and we can reuse the dependencies already installed by the lint job.
Or why would you separate them @david-iov?

---

Ok it does make sense to keep them in the same job. You can call the job CI. That's probably a better name.